classes/Cart.php

```diff
@@ -761,30 +761,30 @@ public function getProducts($refresh = false, $id_product = false, $id_country =
         $sql->leftJoin('image_shop', 'image_shop', 'image_shop.`id_product` = p.`id_product` AND image_shop.cover=1 AND image_shop.id_shop=' . (int) $this->id_shop);
         $sql->leftJoin('image_lang', 'il', 'il.`id_image` = image_shop.`id_image` AND il.`id_lang` = ' . (int) $this->getAssociatedLanguage()->getId());
 
-        $result = Db::getInstance()->executeS($sql);
+        $products = Db::getInstance()->executeS($sql);
```

Comment:
To simplify the review, I think you can do this in another PR, because it not related I think.

Reply:
I moved those commits here: #27416